opentelemetry-distro/README.rst

```diff
@@ -0,0 +1,111 @@
+OpenTelemetry Distro
```

Comment:
Should the distro for the default sdk exist inside the `opentelemetry-instrumentation` package?

Reply:
I wouldn't expect the `opentelemetry-instrumentation` to configure a default distro, in the sense that they serve different purposes, the instrumentation package provides the API and tools to provide auto-instrumentation functionality. Where as the distro package makes decisions for users of what the default configuration should be when run in combination with the `opentelemetry-instrument` command.

Reply:
That's fair. Having more packages is always a point of contention for users. We will probably have to have explicit and robust documentation on how to use these.